Testscripts/Linux/SetupRDMA.sh

```diff
@@ -470,7 +517,6 @@ function Main() {
 		cd mpi-benchmarks/src_c
 		LogMsg "Building Intel MPI Benchmarks tests"
 		make -j $(nproc)
-		Verify_Result
```

Comment:
why remove this?

Reply:
Fixed in 66ae117079544b049a7a20e2d236a40d7b8e8825